styles/scss/utils/_variable.scss

```diff
@@ -35,7 +36,7 @@ $itemMinHeightMd: 30rem;
 $sideBarWidth: 15rem;
 $mobileSpacing: 1.5rem;
 $borderRadius: 0.8rem;
-$settingPageWidth: 105rem;
```

Comment:
Why we need to delete these variables?

Reply:
I moved it from line38 to line50. Make relative variables together.